src/main/js/siblings.js

```diff
@@ -28,37 +28,54 @@
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-/*globals $:false, document:false, window:false */
+document.addEventListener('DOMContentReady', function () {
+  var DELTA = 600;
 
-$(document).ready(
-  function () {
-    "use strict";
-    $(window).scroll(
-      function () {
-        var $box = $('#talks'), more = $box.attr('data-more'), $tail = $('#tail');
-        if ($(window).scrollTop() >= $(document).height() - $(window).height() - 600 && more) {
-          $box.removeAttr('data-more');
-          $tail.show();
-          $.ajax(
-            {
-              url: more,
-              cache: false,
-              dataType: 'xml',
-              headers: { 'Accept': 'text/html' },
-              method: 'GET',
-              success: function (data) {
-                var $div = $(data).find('#talks');
-                $box.html($box.html() + $div.html());
-                $box.attr('data-more', $div.attr('data-more'));
-                $tail.hide();
-              },
-              error: function () {
-                $tail.html('Oops, an error :( Please, try to reload the page');
-              }
-            }
-          );
+  function needMore () {
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/yegor256/rultor/pullRequest?prid=9694263)
```suggestion
  function needMore() {
```